infra/infra-queue/infra-queue-consumer/src/main/java/com/sflpro/notifier/queue/consumer/notification/push/KafkaConsumerListenerServiceImpl.java

```diff
@@ -0,0 +1,30 @@
+package com.sflpro.notifier.queue.consumer.notification.push;
+
+import com.sflpro.notifier.queue.amqp.rpc.RPCQueueMessageHandler;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.kafka.annotation.KafkaListener;
+import org.springframework.stereotype.Service;
+
+/**
+ * Company: SFL LLC
+ * Created on 06/02/2018
+ *
+ * @author William Arustamyan
+ */
+@Service
+@ConditionalOnProperty(name = "queue.engine", havingValue = "kafka")
+public class KafkaConsumerListenerServiceImpl {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(KafkaConsumerListenerServiceImpl.class);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "LOGGER" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1068)